specification/loadtestservice/LoadTestService.Management/models.tsp

```diff
@@ -0,0 +1,258 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+using Azure.ResourceManager.Private;
+
+namespace Microsoft.LoadTestService;
+
+@doc("LoadTest details.")
+model LoadTestResource is TrackedResource<LoadTestProperties> {
+  @doc("Load Test name")
+  @key("loadTestName")
+  @extension("x-ms-parameter-location", "method")
+  @path
+  @segment("loadTests")
+  name: string;
+
+  ...ManagedServiceIdentity;
+}
+
+@doc("Resources provisioning states.")
+enum ResourceState {
+  @doc("Deleted state.")
+  Deleted: "Deleted",
+
+  ...ResourceProvisioningState,
+}
+
+// can be renamed but since the type is already used in published docs, it will be a breaking change.
+@doc("Managed identity type to use for accessing encryption key Url.")
+enum type {
+  @doc("System assigned identity.")
+  SystemAssigned: "SystemAssigned",
+
+  @doc("User assigned identity.")
+  UserAssigned: "UserAssigned",
+}
+
+@doc("Details about the connection between the Batch service and the endpoint.")
+model EndpointDetail {
+  @visibility("read")
+  @doc("The port an endpoint is connected to.")
+  port?: int32;
+}
+
+@doc("A domain name and connection details used to access a dependency.")
+model EndpointDependency {
+  @doc("The domain name of the dependency. Domain names may be fully qualified or may contain a * wildcard.")
+  @visibility("read")
+  domainName?: string;
+
+  @doc("Human-readable supplemental information about the dependency and when it is applicable.")
+  @visibility("read")
+  description?: string;
+
+  @doc("The list of connection details for this endpoint.")
+  @visibility("read")
+  @extension("x-ms-identifiers", ["port"])
+  endpointDetails?: EndpointDetail[];
+}
+
+@doc("A collection of related endpoints from the same service for which the Batch service requires outbound access.")
+model OutboundEnvironmentEndpoint {
+  @doc("The type of service that Azure Load Testing connects to.")
+  @visibility("read")
+  category?: string;
+
+  @doc("The endpoints for this service to which the Batch service makes outbound calls.")
+  @visibility("read")
+  @extension("x-ms-identifiers", ["domainName"])
```

Comment:
this is not generating the expected result. It is giving `"x-ms-identifiers": []` instead of `"x-ms-identifiers": ["domainName"]`  [Example](https://azure.github.io/typespec-azure/playground?c=aW1wb3J0ICJAdHlwZXNwZWMvaHR0cCI7CtIZcmVzdNUZdmVyc2lvbmluZ8wfYXp1cmUtdG9vbHMvyCstxhVjb3Jl3yvIK3Jlc291cmNlLW1hbmFnZXIiOwoKdXNpbmcgVHlwZVNwZWMuSHR0cDvQFVJlc3TRFVbpAKnIG0HEZS5Db3JlzhJSx3dNxnY7CgovKiogQ29udG9zbyDIHiBQcm92aWRlciDmAJ5tZW50IEFQSS4gKi8KQGFybcggTmFtZXNwYWNlCkBzZXJ2aWNlKHsKICB0aXRsZTogIsdYyC5IdWJDbGllbnQiLAp9KQpA5wFgZWQo5wDBcykKbshSIE1pY3Jvc29mdC7SR%2B8AuEFQSSDHTXPkAKNlbnVtIMhUIOQAksQuMjAyMS0xMC0wMS1wcmV2aWV3yDXENCAgQHVzZURlcGVuZGVuY3ko9QEx6AFrcy52MV8wX1DGSF8xKdhA5AGD1zUyxTVhcm1Db21tb27kAc5zxyrXfcspy1Q1xEhg8gDeYCwKfeYBKEHoASrrAUMg6AKC5AD9bW9kZWwgRW1wbG95ZWUgaXMgVHJhY2tlZOgAgjzIHFByb3BlcnRpZXM%2B6QFZ5AIGIG9mIGXIQOYBT3BhdHRlcm4oIl5bYS16QS1aMC05LV17MywyNH0kIuUA%2FWtleSgiyDfER8YXcGF0aMQIc2Vn5AJ7yiNzxSDkAiQ6IHN0cuUC8ecA5OkAvXDpAKLyANrqAL7pAL1BZ%2FMAvGFnZT86IGludDMyOwrHKUNpdHnSKmNpdHk%2F7QCNxnJBxGDkAOAKICBkb21haW7kAN%2FTKULEKWVuZHBvaW50RGV0YWlscz86QVtdxSRpbnRlcmbkAw9PcGVyYXTlAtFleHRlbmRz9gPTLsspe30K5QOvyCPKG8tZ6AEc5gETZ2V05AH%2FQco1UmVhZOkB%2FyzkBCRwb25zZSA9IEI%2BO%2BQBAnJlYXRlT3JVcGRhdOUCPMs9Q8cdUmVwbGFjZUFzeW5jyU3FP%2BQA5g%3D%3D&e=%40azure-tools%2Ftypespec-autorest&options=%7B%7D)